debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbSnapshotChangeEventSource.java

```diff
@@ -217,7 +217,7 @@ public SnapshottingTask getSnapshottingTask(MongoDbPartition partition, MongoDbO
     }
 
     @Override
-    protected SnapshotContext<MongoDbPartition, MongoDbOffsetContext> prepare(MongoDbPartition partition) {
+    protected SnapshotContext<MongoDbPartition, MongoDbOffsetContext> prepare(MongoDbPartition partition, boolean onDemand) {
```

Comment:
Are monogo/oracle changes needed ? Is there some compilation issue which is the motivation for this ?

Reply:
Not necessarily required. Added to handle the merge conflict that had a change across all DBZ connectors.
This change has no impact anyway.